### DISCUSSION TITLE
How colud I suspend a running workflow?

### DISCUSSION THREAD

**Discussion by AngrySKL** [danielgerlag/workflow-core#1221] [Q&A]
I have a running workflow, I want to suspend it during the running, but the call to 'workflowhost.Suspendworkflow' always return false, anyone knows how to make this happen?

**Comment by AngrySKL** [comment]
I think i get the idea, you can first use waitfor to make the workflowcore wait until certain event is triggered, and then at that moment, the lock is released then you can call suspend and resume. You can not suspend a running workflow. Am i right? @danielgerlag

**Reply by MOwaisfarooq** [reply]
You can suspend the running workflow
